x-pack/plugins/observability_ai_assistant/public/components/insight/insight_base.tsx

```diff
@@ -66,10 +69,18 @@ export function InsightBase({
               <AssistantAvatar size="xs" />
             </EuiFlexItem>
             <EuiFlexItem>
-              <EuiText css={{ marginTop: 2, marginBottom: 1 }}>
-                <h5>{title}</h5>
-              </EuiText>
+              <EuiFlexGroup gutterSize="s" alignItems="center">
+                <EuiText css={{ marginTop: 2, marginBottom: 1 }}>
```

Comment:
Was already in the old version, but `EuiText` is meant to take a string as a child component, it adds a `<p>` itself. 

It's a little confusing because `EuiTitle` _does_ require a `<h1 | h2 | h3 | etc>` child component.

Reply:
Is this unreleased change because the docs do have `<h1>` within EuiText https://eui.elastic.co/#/display/text

```
export default () => (
  <div>
    <EuiText grow={false}>
      <h1>This is Heading One</h1>
      ...
 ```

Reply:
Hmm, I stand corrected! I could've sworn I saw console warnings before that would be fixed by removing child wrapping elements. But you are right, this is indeed possible.